mock/mock_test.go

```diff
@@ -1204,6 +1204,18 @@ func Test_Arguments_Diff_DifferentNumberOfArgs(t *testing.T) {
 
 }
 
+func Test_Arguments_Diff_DifferentNumberOfArgsWithAnything(t *testing.T) {
+
+	var args = Arguments([]interface{}{Anything})
+	var diff string
+	var count int
+	diff, count = args.Diff([]interface{}{})
+
+	assert.Equal(t, 1, count)
+	assert.Contains(t, diff, `(Missing) != mock.Anything`)
+
+}
+
```

Comment:
You could add a test case for `Arguments([]interface{}{}).Diff(Arguments([]interface{}{Anything}))`, expected is missing